frontend/public/components/modals/cluster-update-modal.tsx

```diff
@@ -84,23 +144,103 @@ const ClusterUpdateModal = withHandlePromise((props: ClusterUpdateModalProps) =>
           <label id="version-label">{t('public~Select new version')}</label>
           <Select
             aria-labelledby="version-label"
-            onToggle={onToggle}
-            onSelect={onSelect}
+            onToggle={onToggleVersion}
+            onSelect={onSelectVersion}
             selections={desiredVersion}
             isOpen={isOpen}
             isDisabled={clusterUpgradeableFalse && !currentMinorVersionPatchUpdate}
           >
             {options}
           </Select>
         </div>
+        <div className="form-group">
+          <div className="form-group">
+            <label>
+              {t('public~Update options')}
+              <FieldLevelHelp>
+                {t(
+                  "public~Full cluster update allows you to update all your nodes, but takes longer. Partial cluster update allows you to pause worker and custom pool Nodes to accommodate your maintenance schedule, but you'll need to resume the non-master node update within 60 days to avoid failure.",
+                )}
+              </FieldLevelHelp>
+            </label>
+            <Radio
+              isChecked={upgradeType === upgradeTypes.Full}
+              name={upgradeTypes.Full}
+              onChange={handleUpgradeTypeChange}
+              label={t('public~Full cluster update')}
+              id={upgradeTypes.Full}
+              value={upgradeTypes.Full}
+              description={t(
+                'public~{{master}}, {{worker}}, and custom pool {{resource}} are updated concurrently. This might take longer, so make sure to allocate enough time for maintenance.',
+                {
+                  master: NodeTypeNames.Master,
+                  worker: NodeTypeNames.Worker,
+                  resource: NodeModel.labelPlural,
+                },
```

Comment:
NodeTypeNames are not translated... any reason we are not making the read values part of the translations? Are these values expected to change? I would think `Master` and `Worker` are idiomatic names and thus not subject to changes.